src/extensions/codeql-autosetup/buildandreleasetask/index.ts

```diff
@@ -63,17 +63,25 @@ async function run() {
 
         language.forEach(lang => {
             console.log(`Language: ${lang.name}`);
+
+
+            // Add just the languages supported by CodeQL
             switch(lang.name){
-                case "Ruby": languages.add("ruby");break;
+                case "C#": languages.add("csharp");break;
+                case "C++": languages.add("cpp");break;
+                case "Java": languages.add("java");break;
+                case "Kotlin": languages.add("java");break;
```

Comment:
Go,CPP,Swift, and Kotlin not currently supported by build mode none so have a lower success rate.

Go should be higher success compared to the rest but still not as high as others.

Reply:
https://github.com/microsoft/GHAzDO-Resources/issues/37